docs/tracee-ebpf/trace-options.md

```diff
@@ -196,3 +196,8 @@ trace all events that originated from bash or from one of the processes spawned
 ```
 --trace comm=bash --trace follow
 ```
+trace the network events over a given interface name
```

Comment:
The convention is to put a blank line before each doc line, so follow the convention.